docs/en_US/TrainingService/SupportTrainingService.md

```diff
@@ -0,0 +1,36 @@
+# Support TrainingService
```

Comment:
Change the title to TrainingService, as this is a page for Training Service root.

Reply:
fixed.